tomee/content.md

```diff
@@ -24,6 +24,14 @@ Below are some press releases that might be helpful:
 -	Tomitribe Announces Enterprise Service Support for Apache TomEE http://www.tomitribe.com/company/press/tomitribe_enterprise_service_support_for_apache_tomee_javaone_2013/
 -	Tomitribe Introduces Community Partnership Program Conference http://www.tomitribe.com/company/press/tomitribe-introduces-community-partnership-program-and-presents-java-ee-sessions-at-javaone-2014-conference/
 
+# Apache TomEE and ManageCat
+
+Another commercial support company provides enterprise support for Apache TomEE is ManageCat. ManageCat involves with a lot of Apache Java EE projects to contribute open source Java EE ecosystem. ManageCat co-founder is also a founder of open source Apache OpenWebBeans project (http://openwebbans.apache.org) which is heavily used in Apache TomEE application server for CDI integration. 
```

Comment:
The CI is failing on the extra space at the end of this line.  You have to run `./markdownfmt.sh -d` to see the diff of changes that markdownfmt requires.